src/waveform/renderers/waveformmarkproperties.h

```diff
@@ -0,0 +1,27 @@
+#ifndef WAVEFORMMARKPROPERTIES_H
+#define WAVEFORMMARKPROPERTIES_H
+
+#include <QColor>
+#include <QDomNode>
+
+class SkinContext;
+class WaveformSignalColors;
+
+class WaveformMarkProperties {
+  public:
+    WaveformMarkProperties();
+    WaveformMarkProperties(const QDomNode& node,
+                           const SkinContext& context,
+                           const WaveformSignalColors& signalColors);
+    virtual ~WaveformMarkProperties();
```

Comment:
WaveformMarkProperties is not intended to be used as a base class. Therefore I would delete the (virtual) destructor entirely and declare the class as 'final'. C++11 strikes again ;)